funnel/views/commentvote.py

```diff
@@ -109,7 +109,7 @@ def new_comment(self):
                                 send_mail_info.append(
                                     {
                                         'to': self.obj.owner.email or self.obj.email,
-                                        'subject': u"[New Comment] {project}: {proposal}".format(
+                                        'subject': u"[💬] {project}: {proposal}".format(
```

Comment:
The brackets are not needed when using emoji. Can you remove them? @iambibhas

Reply:
Removed them. I kept them because otherwise it seems a bit lacking in clues of these being emails about comments. The bubble right beside the project name and proposal name seems so tiny.